src/storhaug

```diff
@@ -399,6 +402,7 @@ setup_create_resources()
             config="${HA_NFS_CONF}" \
             state_fs="${HA_NFS_TYPE}" \
             state_mnt="${HA_MNT_DIR}/${HA_NFS_MNT_DIR}" \
+        op monitor on-fail="block" \
```

Comment:
I don't think it will restart indefinitely. If I remember right, there is a default number of times pacemaker will try to restart a resource before it declares it broken.